### ISSUE TITLE
UnboundLocalError: local variable 'indentation' referenced before assignment

### ISSUE THREAD

**Issue by Zac-HD** [asottile/blacken-docs#68]
```console
$ blacken-docs t.rst 
Traceback (most recent call last):
  File "/tmp/c/venv/bin/blacken-docs", line 8, in <module>
    sys.exit(main())
  File "/tmp/c/venv/lib/python3.8/site-packages/blacken_docs.py", line 210, in main
    retv |= format_file(filename, black_mode, skip_errors=args.skip_errors)
  File "/tmp/c/venv/lib/python3.8/site-packages/blacken_docs.py", line 166, in format_file
    new_contents, errors = format_str(contents, black_mode)
  File "/tmp/c/venv/lib/python3.8/site-packages/blacken_docs.py", line 155, in format_str
    src = RST_PYCON_RE.sub(_rst_pycon_match, src)
  File "/tmp/c/venv/lib/python3.8/site-packages/blacken_docs.py", line 139, in _rst_pycon_match
    code += orig_line[indentation:] + '\n'
UnboundLocalError: local variable 'indentation' referenced before assignment
```

```rst
.. code-block:: pycon

    pass
```

This is clearly bad in that it's not actually a console session (no leading `>>>`), but it would be better to emit a warning and skip the block than to crash with an internal error.

**Comment by asottile** [comment]
@jdufresne ^

**Comment by jdufresne** [comment]
Thanks for the report. Fixed in #70.

FWIW, the example `pycon` block isn't really a Python console as there is no `>>>` prompt. But either way, a crash isn't good.

**Comment by Zac-HD** [comment]
Cheers!  Agreed that the document is clearly wrong, but also that it's better not to crash on bad documents 😁
